elliottlib/bzutil.py

```diff
@@ -297,7 +277,7 @@ def creation_time_parsed(self):
         return datetime.strptime(str(self.bug.fields.created), '%Y-%m-%dT%H:%M:%S.%f%z')
 
     def is_ocp_bug(self):
-        return self.bug.fields.project.key == "OCPBUGS"
+        return self.bug.fields.project.key == "OCPBUGS" and 'Placeholder' not in self.summary
```

Comment:
There is a non-zero chance that a real bug exists with `Placeholder` in the title. Maybe this can be narrowed down with asserting that `Component` is `Release` and `Labels` contains `Automation`?

Reply:
Added